modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/snapshot/CurrencySnapshotValidator.scala

```diff
@@ -156,6 +158,10 @@ object CurrencySnapshotValidator {
                 case None =>
                   creationResult.focus(_.artifact.dataApplication).replace(expected.dataApplication)
               }
+            }.map { creationResult =>
+              if (creationResult.artifact.messages.forall(_.isEmpty))
+                creationResult.focus(_.artifact.messages).replace(expected.messages)
+              else creationResult
```

Comment:
Do we still need this? This was brought over when the branch was first created and it's not in develop now.